### ISSUE TITLE
[Feat]: Support IntelliJ Idea 2023.3.2

### ISSUE THREAD

**Issue by tbm98** [Welltested-AI/fluttergpt#174]

**Comment by doitwithdk** [comment]
@tbm98 Appreciate your feature suggestion! Good news – we've got IntelliJ support in the works and are hammering out the initial setup behind the scenes. If you're keen to jump in and contribute, we'd love to have your expertise onboard. Feel free to reach out if you're interested or have any more awesome ideas! 🚀

**Comment by IldySilva** [comment]
@doitwithdk  i'would  like to contribute on this feature,What stage are we at?

**Comment by samyakkkk** [comment]
Hi @IldySilva, thanks for your interest in contributing. We should be rolling out a ticket for this by end of upcoming week.

**Comment by neiljaywarner** [comment]
@samyakkkk @IldySilva  I'm looking forward to this, is there a way i can use it tomorrow even if in alpha or beta?

**Comment by samyakkkk** [comment]
@neiljaywarner we really appreciate your interest, but unfortunately, the IntelliJ version still has sometime before it becomes ready to be used in beta. 

We'll keep everyone posted in thread and are working hard to bring it at the soonest. Thanks.

**Comment by neiljaywarner** [comment]
thanks
i tried the vscode version and it didn't seem that easy to me,but again,
i'm not used to vscode palette etc.

Not sure if you want to make a separate ticket, but are you working on more
documentation? in particular I'd like to know if you support riverpod and
more about create from blueprint; i didn't even see create from blueprint
in vscode last time i looked :(

**Comment by neiljaywarner** [comment]
@samyakkkk  is there a way we could receive notice through email when intellij/android studio one works?
So far commanddash/fluttergpt has been the only big reason for me to switch, but there's still so many things to like about jetbrains/intellij ;) (retrofit tool, easy file new project, double shift, git compare.. of course maybe it is all jsut what you're used to..)

**Comment by samyakkkk** [comment]
@neiljaywarner I'll definitely update you in this thread as soon as the IntelliJ extension comes out. You can also join our community slack - https://join.slack.com/t/welltested-ai/shared_invite/zt-25u09fty8-gaggH9HbmopB~4tialTrlA

Meanwhile I'd love to understand what is your most used feature in CommandDash. In your last thread you mentioned an issue about code from blueprint, which we deprecated and instead is something that could be done with our chat. 

Please let me know if I can assist with anything here. thanks.

**Comment by neiljaywarner** [comment]
I didn't realize it was deprecated I apologize.

From my perspective it is extremely difficult to tell if AI in Dev is worth
"messing with".. the value proposition of saving time vs friction and
learning curve.

I get excited but I'm having a hard time seeing where it fits.  I know a
lot of it is me and you guys are doing awesome work. I'm especially
interested in trying to figure out automated testing. I would be thrilled
if AI could generate integration and widget tests with shared code based on
the UI elements. I get the impression a good bit of it you've already done
and I need to explore

Thanks again for your good work and I look forward to seeing how it goes

Does it support riverpod?
Are there any docs or examples how blueprint works please?

Also. How about http or dio or retrofit plus repository better yet from URL?


....sent from my phone

**Comment by samyakkkk** [comment]
@neiljaywarner yes, the AI tools are evolving pretty fast and it is definitely hard when something you use gets deprecated. 

That said, we have kind of become stable now and will maintain our current features. 

For your testing needs, we have a specific @test agent with `/unit /widget and /integration` commands to help you generate all kinds of tests. Here is a doc on how you can use it - https://www.commanddash.io/blog/exploring-the-test-agent

We also have an agent for riverpod, that you can install from the marketplace.

![](https://github.com/CommandDash/packages/raw/main/dash_agent/assets/marketplace-icon.png)

**Comment by neiljaywarner** [comment]
Awesome, thanks for the info!

....sent from my phone
